src/components/Info/Info.style.js

```diff
@@ -1,13 +1,52 @@
 import styled from 'styled-components';
 
-const InfoWrapper = styled.section`
+const InfoWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 348px;
+  max-width: 738px;
+  margin: 130px 0 100px 0;
+  @media (max-width: 768px) {
+    margin: 30px 0 30px 0;
+  }
+`;
+
+const InfoSection = styled.section`
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
-  height: 100%;
   width: 100%;
-  max-width: 738px;
-  text-align: left;
+  line-height: 27px;
+  @media (max-width: 768px) {
+    margin-top: 30px;
+    padding: 0 20px;
+  }
+`;
+
+const InfoHeading = styled.h2`
+  margin: 0 0 13px 0;
+`;
+
+const InfoP = styled.p`
+  margin: 0;
```

Comment:
If you like you can set the `letter-spacing: 0.03px;` You'll see that for long strings this is a bit better. Might also be a global style